app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = current_user.comments.new(post_id: @post.id, author_id: current_user.id, text: comment_params)
```

Comment:
- [ ] there is an error about your use of author_id, your schema has user_id, so you either change all author_id to user id or change the user_id in your schema to author_id
![Screenshot from 2022-08-19 19-49-43](https://user-images.githubusercontent.com/66097371/185688983-fea0f44f-03c4-449a-acfc-30be65e2bbc1.png)

Reply:
Hi @VICTOR-CHUKWU I am having an issue here, the project is running well on my side and when I push it for a review you guys get an error, would you please have a zoom call with me and help me. I am on last review and I can't change an error I am not having on this end.

Reply:
I hope you are clear now with my explanations. nice work tho :+1: